src/python/pants/backend/python/goals/export_test.py

```diff
@@ -36,6 +47,10 @@ def rule_runner() -> RuleRunner:
             *target_types_rules.rules(),
             *distdir.rules(),
             *local_dists_pep660.rules(),
+            *isort_subsystem.rules(),  # add a tool that we can try exporting
+            UnionRule(
+                ExportableTool, isort_subsystem.Isort
+            ),  # TODO: remove this manual export when we add ExportableTool to tools
```

Comment:
What's the status of this TODO?

Reply:
Oh, NM, it'll happen once we actually use this for isort. I assume you'll remember... :)